pkgs/by-name/pl/plakativ/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+  guiSupport ? true,
+}:
+let
+  pname = "plakativ";
+  version = "0.5.2";
+in
+python3Packages.buildPythonApplication {
+  inherit pname version;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-pRRx/cBgf+HK9NZ1NnakL/dbxGn1UNHJOGvC3I70QGI=";
+  };
+
+  dependencies =
+    with python3Packages;
+    [
+      pymupdf
+    ]
+    ++ lib.optional guiSupport tkinter;
+
+  meta = {
+    description = "Convert a PDF into a large poster that can be printed on multiple smaller pages";
+    mainProgram = "plakativ";
+    homepage = "https://gitlab.mister-muffin.de/josch/plakativ";
+    license = lib.licenses.gpl3;
```

The file header doesn't mention "or any later version".

```suggestion
    license = lib.licenses.gpl3Only;
```